ee/vellum_ee/workflows/display/tests/workflow_serialization/test_basic_api_node_serialization.py

```diff
@@ -54,7 +54,6 @@ def test_serialize_workflow(vellum_client):
 
     # AND its raw data should be what we expect
     workflow_raw_data = serialized_workflow["workflow_raw_data"]
-    assert workflow_raw_data.keys() == {"edges", "nodes", "display_data", "definition"}
```

Comment:
Each of the following tests are ensuring serialization for a node - so this assertion isn't relevant. It's already covered by our workflow level tests.

The same could be said for a lot of the other assertions in these tests, but wanted to keep the PR focused